src/redux/catefories/categories.js

```diff
@@ -0,0 +1,10 @@
+export default function categoriesReducer(state = [], action) {
+  switch (action.type) {
+    case 'CHECK_STATUS':
+      return 'Under construction';
+    default:
+      return state;
+  }
+}
+
+export const status = () => ({ type: 'CHECK_STATUS' });
```

Comment:
Kindly make sure the  value of the action type also follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`